rebar.config

```diff
@@ -8,5 +8,5 @@
 
 {deps_dir, "deps"}.
 {deps, [
-        {lager, ".*", {git, "https://github.com/MiniclipPortugal/lager.git", {tag, "2.1.1"}}}
+        {lager, ".*", {git, "https://github.com/erlang-lager/lager.git", {tag, "3.2.4"}}}
```

Comment:
If I'm reading it correctly, we're importing lager so that we can log [this single situation](https://github.com/MiniclipPortugal/lhttpc/blob/master/src/lhttpc_manager.erl#L335). Does this still happen? It looks like the mainstream lhttpc version doesn't depend upon lager - it was me who [added it](https://github.com/g-andrade/lhttpc/commit/2f4dc89d6ce815fd25f7155cc648b1abad489008). Perhaps it's no longer needed?